open-metadata-implementation/access-services/data-engine/data-engine-server/src/main/java/org/odpi/openmetadata/accessservices/dataengine/server/mappers/SoftwareServerPropertiesMapper.java

```diff
@@ -3,8 +3,8 @@
 package org.odpi.openmetadata.accessservices.dataengine.server.mappers;
 
 /**
- * SoftwareServerPropertiesMapper provides property name mapping for Software Servers. Specifically, a single engine
- * is represented as an SoftwareServerCapabilityRequestBody entity.
+ * SoftwareServerPropertiesMapper provides property name mapping for external data engines. Specifically, a single data
+ * engine is represented as an DataEngineRegistrationRequestBody entity.
  */
 public class SoftwareServerPropertiesMapper {
```

Comment:
Should we rename this class to be consistent with the other renamings (from `SoftwareServerPropertiesMapper` to `DataEnginePropertiesMapper`) (?)

Reply:
True. I will rename this part. :)